src/components/data-management/SamplesTable.jsx

```diff
@@ -254,71 +261,20 @@ const SamplesTable = forwardRef((props, ref) => {
     setTableData(newData);
   }, [projects, samples, activeProjectUuid]);
 
-  // Name of file is 1.Example-File_Name.zip
-  // The naming starts from 1
-  const parseFileName = (name) => {
-    const parts = name.split('.');
-
-    // The first part of the file name is the order
-    const order = Number.parseInt(parts[0], 10);
-    if (!order) {
-      console.warn('Invalid dataset file:', name);
-      return null;
-    }
-
-    // Filename may contain dots, so we need to join them
-    const rawFileName = parts.slice(1, parts.length - 1).join('.');
-    const filename = rawFileName.replace(/_/g, ' ');
-
-    return { order, filename };
-  };
-
-  const getPublicDatasets = async () => {
-    const accountId = getAccountId(environment);
-    Storage.configure({
-      bucket: `biomage-public-datasets-${environment}-${accountId}`,
-      customPrefix: {
-        public: '',
-      },
-    });
-
-    const datasets = await Storage.list('');
-
-    // Properly order the datasets
-    const result = new Array(datasets.length).fill(null);
-
-    datasets.forEach((data) => {
-      const parsed = parseFileName(data.key);
-
-      if (!parsed) return;
-
-      // Order begins from 1, but array index begins from 0
-      const insertionIndex = parsed.order - 1;
-      result[insertionIndex] = {
-        filename: data.key,
-        description: parsed.filename,
-      };
-    });
-
-    return result.filter((data) => data !== null);
-  };
-
   useEffect(() => {
     if (!environment) return;
 
-    getPublicDatasets().then((datasets) => { setExampleDatasets(datasets); });
+    fetchAPI('/v2/experiments/examples').then((experiments) => {
+      setExampleExperiments(experiments);
+    });
   }, [environment]);
 
-  const downloadPublicDataset = async (filename) => {
-    const accountId = getAccountId(environment);
-    const s3Object = await Storage.get(
-      filename,
-      {
-        bucket: `biomage-public-datasets-${environment}-${accountId}`,
-        contentType: 'multipart/form-data',
-      },
-    );
-    downloadFromUrl(s3Object);
+  const cloneIntoCurrentExperiment = async (exampleExperimentId) => {
+    const url = `/v2/experiments/${exampleExperimentId}/clone/${activeProjectUuid}`;
```

Comment:
With v2, we have projectUuid == experimentId. Is it difficult to rename/refactor `activeProjectUuid` to `activeExperimentId`?